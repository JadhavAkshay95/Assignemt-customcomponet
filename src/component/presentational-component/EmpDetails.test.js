import { shallow } from "enzyme";
import { EmpDetails } from "./EmpDetails";
import React from "react";

const employee = {
  id: "1",
  name: "John Smith",
  designation: "Senior UI developer",
  exp: 11,
  technologies: ["XD", "Sketch", "Figma"],
  skills: ["UX", "UI", "Visual"],
  avaibility: "3",
  location: ["Indore", "Pune"],
  isOpenForRelocation: false,
  message: "",
};

describe("Emp details Component", () => {
  it("Should render Emp details component", () => {
    let open = "true";
    const wrapper = shallow(<EmpDetails employee={employee} open={open} />);
    expect(wrapper).toMatchSnapshot();
  });
});
