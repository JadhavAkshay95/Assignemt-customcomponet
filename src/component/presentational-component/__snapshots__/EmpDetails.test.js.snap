// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Emp details Component Should render Emp details component 1`] = `
<UNDEFINED
  employee={
    Object {
      "avaibility": "3",
      "designation": "Senior UI developer",
      "exp": 11,
      "id": "1",
      "isOpenForRelocation": false,
      "location": Array [
        "Indore",
        "Pune",
      ],
      "message": "",
      "name": "John Smith",
      "skills": Array [
        "UX",
        "UI",
        "Visual",
      ],
      "technologies": Array [
        "XD",
        "Sketch",
        "Figma",
      ],
    }
  }
  open="true"
/>
`;
